internal/driver/controller.go

```diff
@@ -20,6 +20,14 @@ import (
 
 var ctrlLogger = ctrl.Log.WithName("driver").WithName("controller")
 
+var (
+	ErrNoNegativeRequestBytes = errors.New("required capacity must not be negative")
+	ErrNoNegativeLimitBytes   = errors.New("capacity limit must not be negative")
+	ErrRequestedExceedsLimit  = errors.New("requested capacity exceeds limit capacity")
+	ErrResultingRequestIsZero = errors.New("requested capacity is 0")
+	ErrExceedsMinimumXfsSize  = fmt.Errorf("limit capacity is less than the minimum size for xfs (%d bytes)", topolvm.MinimumXfsSize)
```

Comment:
`ErrSubceedsMinimumXfsSize`?

Reply:
In order to keep things okay for non-native speakers I chose `ErrLimitLessThanMinimumXfsSize` as its much easier to understand.

Reply:
awesome